fuzz/sst_file_writer_fuzzer.cc

```diff
@@ -62,27 +91,31 @@ DEFINE_PROTO_FUZZER(DBOperations& input) {
   rocksdb::SstFileWriter writer(env_options, options);
   rocksdb::Status s = writer.Open(sstfile);
   CHECK_OK(s);
+  std::map<std::string, std::string> kv;
   for (const DBOperation& op : input.operations()) {
     switch (op.type()) {
-      case OpType::PUT:
+      case OpType::PUT: {
         s = writer.Put(op.key(), op.value());
         CHECK_OK(s);
+        kv[op.key()] = op.value();
         break;
-      case OpType::MERGE:
-        s = writer.Merge(op.key(), op.value());
-        CHECK_OK(s);
-        break;
-      case OpType::DELETE:
+      }
+      case OpType::DELETE: {
         s = writer.Delete(op.key());
         CHECK_OK(s);
+        kv.erase(op.key());
```

Comment:
I thought SstFileReader just return each record put, delete or merge. Does it apply delete?